src/app/snark_work_debugger/snark_work_debugger.ml

```diff
@@ -0,0 +1,57 @@
+open Core
```

Comment:
This file is deleted in `compatible`. Somehow it's back. We need to remove it again.